docs/source/algorithms.md

```diff
@@ -565,6 +565,222 @@ you install estimagic.
+.. dropdown::  scipy_differential_evolution
+
+    .. code-block::
+
+        "scipy_differential_evolution"
+
+    Find the global minimum of a multivariate function using differential evolution (DE). DE is a gradient-free method.
+
+    Due to estimagic's general parameter format the integrality and vectorized
+    arguments are not supported.
+
+    The algorithm supports the following options:
+
+    - **strategy** (str): Measure of quality to improve a candidate solution, can be one of the following keywords
+      - ‘best1bin’
+      - ‘best1exp’
+      - ‘rand1exp’
+      - ‘randtobest1exp’
+      - ‘currenttobest1exp’
+      - ‘best2exp’
+      - ‘rand2exp’
+      - ‘randtobest1bin’
+      - ‘currenttobest1bin’
+      - ‘best2bin’
+      - ‘rand2bin’
+      - ‘rand1bin’
+    ,default is 'best1bin'.
+    - **stopping.max_iterations** (int): The maximum number of criterion evaluations without polishing is(stopping.max_iterations + 1) * population_size * number of parameters)
+    - **population_size_multiplier** (int): A multiplier setting the population size. The number of individuals in the population is population_size * number of parameters. The default 15.
+    - **convergence.relative_criterion_tolerance** (float): Default 0.01.
+    - **mutation_constant** (float/tuple): The differential weight denoted by F in literature. Should be within 0 and 2.  The tuple form is used to specify (min, max) dithering which can help speed convergence.  Default is (0.5, 1).
+    - **recombination_constant** (float): The crossover probability or CR in the literature determines the probability that two solution vectors will be combined to produce a new solution vector. Should be between 0 and 1. The default is 0.7.
+    - **seed** (int): DE is stochastic. Define a seed for reproducability.
+    - **polish** (bool): Uses scipy's L-BFGS-B for unconstrained problems and trust-constr for constrained problems to slightly improve the minimization. Default is True.
+    - **sampling_method** (str/np.array): Specify the sampling method for the initial population. It can be one of the following options
+       - "latinhypercube"
+       - "sobol"
+       - "halton"
+       - "random"
+       - an array specifying the initial population of shape (total population size, number of parameters). The initial population is clipped to bounds before use. Default is 'latinhypercube'
+    - **convergence.absolute_criterion_tolerance** (float): CONVERGENCE_SECOND_BEST_ABSOLUTE_CRITERION_TOLERANCE
+    - **n_cores** (int): The number of cores on which the function is evaluated in
+    parallel. Default 1.
+    - **batch_evaluator** (str or callable). An estimagic batch evaluator. Default
+    'joblib'.
+
+```
+
+
+```{eval-rst}
+.. dropdown::  scipy_shgo
+
+    .. code-block::
+
+        "scipy_shgo"
+
+    Find the global minimum of a fuction using simplicial homology global optimization.
+
+    Note: scipy_shgo requires the constraints' codomain to be a scalar.
```

Comment:
This is not true anymore. We transform vector-valued constraints into a list of scalar-valued constraints for this algorithm.